doc/effects.md

```diff
@@ -86,6 +86,26 @@ myComponent.addEffect(sequence);
 ```
 An example of how to use the SequenceEffect can be found [here](/doc/examples/effects/sequence_effect).
  
+## CombinedEffect
```

Comment:
I thought of suggesting this when I saw SequenceEffect, I thought it could have a boolean to tell if it's simultaneous or not. But then I thought, you can already add multiple effects to the component and they run at the same time right? does this add anything new? I guess only if you are adding a CombinedEffect inside a SequenceEffect? (just want to understand the use case)

Reply:
The offset would be more work to handle manually. And it makes it a lot easier to make it run it in in different loops with isInfinite and isAlternate. Ultimately you'd want to have different offsets on each, but I couldn't make that implementation look as nice.